website/pages/buy/file-extended.mdx

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Not sure if I made a mistake or if this is something that changed in the last week or so, but I thought these files don't need to be committed because they're generated as part of the prebuild step?

Reply:
Discussed in https://github.com/cloudquery/cloudquery/pull/12239. The `buy` files have a manually generated file too now, so until we separate it from the auto generated ones we need to commit them